lib/src/widgets/animated_button.dart

```diff
@@ -130,6 +126,40 @@ class _AnimatedButtonState extends State<AnimatedButton>
     }
   }
 
+  // initializes width and size animation
+  void _initWidth(BuildContext context) {
+    final theme = Theme.of(context);
+    final fontSize = theme.textTheme.button.fontSize;
+
+    var renderParagraph = RenderParagraph(
+      TextSpan(
+        text: widget.text,
+        style: TextStyle(
+          fontSize: fontSize,
+          fontWeight: theme.textTheme.button.fontWeight,
+          letterSpacing: theme.textTheme.button.letterSpacing,
+        ),
+      ),
+      textDirection: TextDirection.ltr,
+      maxLines: 1,
+    );
+
+    renderParagraph.layout(BoxConstraints(minWidth: 120.0));
+
+    // text width based on fontSize, plus 24.0 for padding
+    var textWidth = renderParagraph.getMinIntrinsicWidth(fontSize).ceilToDouble() + 24.0;
```

Comment:
24.0 is a too small. I suggest 64.0 for padding
![Screenshot_20191205-031935](https://user-images.githubusercontent.com/23303775/70178112-3c7c9500-170e-11ea-87b4-da36bfa1318b.jpg)

Reply:
Thanks. I'll push the updates, but 64.0 padding seems it might be a little too much for lower resolution screens? Here's an example:
<img width="356" alt="Screen Shot 2019-12-04 at 10 58 03 PM" src="https://user-images.githubusercontent.com/55109813/70205537-f0b7f480-16e9-11ea-99d5-e91071a74810.png">